{{cookiecutter.module_name}}/.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: docker
```

Comment:
Shouldn't this be spaced in?

Reply:
it's technically optional in yaml, but sure, done. this file was autogenerated by github for another project, and i had just copied it over...